segment:
  frame_id: "world"
  thickness: 0.02
  color:
    r: 0.0
    g: 0.0
    b: 1.0
    a: 1.0
  point_a: [-0.193508, -0.317403, -0.217864]
  point_b: [0.447289, -0.0610843,  -0.445703]
   
